### DISCUSSION TITLE
How to work with sessions?

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#182] [Ideas]
Do you have a plan to make a tutorial on how to work with sessions?

**Comment by FeodorFitsner** [comment]
Could you please elaborate on your use please?

**Reply by nascin** [reply]
I would like to know how to work with login/logout, session time.

**Comment by FeodorFitsner** [comment]
login/logout is "Authentication" which will be implemented as a next step. Please subscribe to this issue: https://github.com/flet-dev/flet/issues/1

Session time is a lifetime of a `Page` instance created for the user before entering `main()` method. It has sliding expiration and by default it's 60 minutes. Can be changed with `FLET_APP_LIFETIME_MINUTES` environment variable.

**Comment by FeodorFitsner** [comment]
Another question that may arise it storing session-specific data. As Flet is client-server framework all session data will be stored on the server. There is no something like `page.session` right now (as a workaround you can assign anything to a `page` instance as it's Python). I've added a new issue to implement that: https://github.com/flet-dev/flet/issues/192

Have I missed anything?

**Reply by nascin** [reply]
I understand perfectly, wait for the implementation. Thank you for the explanation.
